bot/processors/pennychat.py

```diff
@@ -34,9 +34,11 @@
 PENNY_CHAT_SHARE = 'penny_chat_share'
 PENNY_CHAT_CAN_ATTEND = 'penny_chat_can_attend'
 PENNY_CHAT_CAN_NOT_ATTEND = 'penny_chat_can_not_attend'
+PENNY_CHAT_VISIBILITY = 'penny_chat_visibility'
 
 PENNY_CHAT_ID = 'penny_chat_id'
 
+global penny_chat_visibility_name
```

Comment:
Why do we need this `global` variable?

Reply:
Oh yeah, not sure why I made that a global variable, that can be updated. I think my initial thought process was using it later down the line would cause an issue but I somehow just realized every use of it is in the same function 😄